ParticleObjects/particle_class.f90

```diff
@@ -56,6 +57,7 @@ module particle_class
     integer(shortInt)          :: cellIdx  = -1     ! Cell idx at the lowest coord level
     integer(shortInt)          :: uniqueID = -1     ! Unique id at the lowest coord level
     integer(shortInt)          :: collisionN = 0    ! Number of collisions
+    real(defReal)              :: collisionE = ZERO ! Collision energy
```

Comment:
Is it possible to keep the `collisionE` out of the `particleState`? Or is it necessary there to work with some tally map interfaces and such?

Reply:
NO it's not, I think I put it there automatically.